### PR TITLE
Italian localization update

### PR DESCRIPTION
**Pull request by dej611**
I've started to catch-up the Italian documentation with the changes since last time.
I'll leave here the list of things done (and those to complete):

* [x] `code-of-conduct.md`
* [x] `contributing.md`
* [x]  ~~`mantaining.md`~~
* [x] `readme.md`
* [x] `docs/common-pitfalls.md`
* [x] `docs/recipes/babelrc.md`
* [x] `docs/recipes/browser-testing.md`
* [x] `docs/recipes/code-coverage.md`
* [x] `docs/recipes/debugging-with-chrome-devtools.md`
* [x] `docs/recipes/debugging-with-vscode.md`
* [x] `docs/recipes/debugging-with-webstorm.md`
* [x] `docs/recipes/endpoint-testing.md`
* [x] `docs/recipes/isolated-mongodb-integration-tests.md`
* [x] `docs/recipes/jspm-systemjs.md`
* [x] `docs/recipes/precompiling-with-webpack.md`
* [x]  `docs/recipes/react.md`
* [x]  `docs/recipes/typescript.md`
* [x]  `docs/recipes/vue.md`
* [x]  `docs/recipes/watch-mode.md`
* [x]  `docs/recipes/when-to-use-plan.md `
* [x] `docs/specs/001 - Improving language support.md`
* [x] Update the compare link with latest revision in `master`
* [ ] Review pass

**Note** do not merge until all the check boxes have been completed.
**Note 2** help is welcome, in particular for reviewing the translation.

### CODE REVIEW

**Comment by sindresorhus** [comment]
// @parro-it In case you have any interest in helping reviewing :)

**Comment by parro-it** [comment]
@sindresorhus I was already thinking to help from my typo PR yesterday :wink:

**Comment by sindresorhus** [comment]
@dej611 Still interested in finishing this up?

**Comment by dej611** [comment]
Sure. Sorry to have disappeared. I'll finish this up. ;)

File: it_IT/code-of-conduct.md [hunk 1]
```diff
@@ -24,20 +24,19 @@ Questi sono alcuni esempi di comportamento che inducono a creare un ambiente pos
...
 
 ## Dove si applica questo codice
 
-Questo Codice di Condotta si applica sia all'interno dello spazio del progetto che in luoghi pubblici dove un individuo rappresenta il progetto o la sua comunità che stia usando un indirizzo di email ufficiale del progetto, scrivendo su di un account ufficiale del progetto o agendo come raprresentante ufficiale ad un evento oline o offline. La definizione puntuale della rappresentanza del progetto potrà essere definità in seguito e resa chiara dai amministratori del progetto.
+Questo Codice di Condotta si applica sia all'interno dello spazio del progetto che in luoghi pubblici dove un individuo che rappresenta il progetto o la sua comunità che utilizzi un indirizzo di email ufficiale del progetto, scrivendo su di un account ufficiale del progetto o agendo come rappresentante ufficiale ad un evento online o offline. La rappresentanza del progetto potrà essere definita in seguito e resa chiara dai amministratori del progetto.
```
Comment (parro-it) [inline it_IT/code-of-conduct.md]: Sembra manchi un verbo nella prima frase?

File: it_IT/docs/common-pitfalls.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+## AVA in Docker
+
+Se usi AVA in Docker nel tuo CI, hai bisogno di impostare le variabili d'ambiente correttamente. Nello specifico aggiungi `-e CI=true` nel comando `docker exec`. Per maggiori informazioni guarda [#751](https://github.com/avajs/ava/issues/751).
+
+AVA usa [is-ci](https://github.com/watson/is-ci) per capise se è eseguito in un ambiente di CI o no, basandosi su [questi variabili](https://github.com/watson/is-ci/blob/master/index.js).
+
+## AVA ed il limite di client connessi
+
+Potresti utilizzare un servizio che permette un numero limitato di connessioni concorrenti. Ad esempio, molte aziende con prodotti Database-as-a-service offrono piani con un numero limitato di clienti connessi nello stesso momento. AVA può superare questi limiti poichè esegue processi multiple, ma questi servizi dovrebbero inviare un errore o incodare la richiesta in questi casi. Se il tuo servizio non si comporta in questo modo, i test potrebbero rimanere in sospeso.
```
Comment (parro-it) [inline it_IT/docs/common-pitfalls.md]: Clienti --> client

**Comment by dej611** [comment]
@parro-it as soon as I'll finish with the bulk translation I will pass to your reviews. ;)

**Comment by parro-it** [comment]
Perfect!

**Comment by dej611** [comment]
@parro-it I've completed the translation for all files now. Updated the link in `readme.md` file too.
Let me know if it's ok for you. ;)

**Comment by dej611** [comment]
While still in review I've updated to the very latest commit so far the translation.
I did also forget to add the translation notes on the new files: they are fixed now.
All the compare links should be up to date now.

**Comment by sindresorhus** [comment]
Thanks for all your hard work on this @dej611 🙌

superhighfive

**Pull request merged**